### ISSUE TITLE
Uncaught ReferenceError after using await and async in src 

### ISSUE THREAD

**Issue by janmarkuslanger** [janmarkuslanger/life#1]
After using await and async the transpiled code logs following error
`Uncaught ReferenceError: regeneratorRuntime is not defined`

**Comment by janmarkuslanger** [comment]
```
{
  "presets": [
    ["env", {
      "targets": {
        "browsers": ["last 2 Chrome versions"]
      },
      modules: false
    }]
  ]
}

```
in .babelrc this code works.

**Comment by janmarkuslanger** [comment]
Fixed in 0.15.19-beta
